docs/source/private/overview.rst

```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
 * Report blinded correlation plots of free energy estimates against experimental data.
 
 Blinded data will be collated alongside any relevant meta analyses and published in an appropriate location.
+
+
+Before you start
+****************
+
+
+Please ensure that you use the same environment used to run the public dataset OpenFE
+benchmarks. Please see our benchmarks specific :ref:`benchmark openfe install instructions <installation>`
+for more details.
+
+
+Phase 1: Simulating Private Benchmark Sets
+******************************************
+
+Dataset Selection
+=================
+
+Each participating industry partner will be expected to select and prepare
+benchmarks sets from their own internal projects.
+
+Our aim is to try to assess how well OpenFE performs on the types of projects
+that are being worked on in industry. You are therefore asked to pick **a minimum of one**
+dataset for this task.
+
+The datasets should:
+
+* Have assay data for each ligand in the set.
+* Represent the types of campaigns you run internally; this can either be fully internal or come from pre-published work.
+* Not be part of any commonly published free energy benchmark sets.
+* Where possible, follow `best practices in benchmark set selection <https://livecomsjournal.org/index.php/livecoms/article/view/v4i1e1497>`_.
+
+You are also **encouraged to avoid** datasets which are known to not work with OpenFE.
+This includes:
+
+* Membrane-containing systems.
+* Ligand series that undergo cyclisation.
+* Ligand series that undergo scaffold hopping (i.e. very small to no conserved core).
+
+Should you end up using a system with known challenges, we would ask you to describe the challenging nature of the dataset upon submission.
+
+
+Dataset Preparation
+===================
+
+How the datasets are prepared is fully up to you. We encourage that use lessons learnt from the
+public set :ref:`input preparation step <input-preparation>` as a guide on how to prepare systems
+for use with OpenFE. We expect that you will have a PDB for your protein, an SDF for your ligands,
+and optionally another SDF for your cofactors.
+
+The only additional requirements for dataset preparations are:
+* Any ligand names should be anonymised. The OpenFE data gathering scripts will assume names to be anonymous.
```

Comment:
How about ". . . The OpenFE data gathering scripts will not modify the existing names."